docs/reference/search/search-your-data/semantic-search-elser.asciidoc

```diff
@@ -216,6 +216,69 @@ weights.
 ----
 // NOTCONSOLE
 
+
+[discrete]
+[[text-expansion-compound-query]]
+=== Combining `text_expansion` with a compound query
```

Comment:
```suggestion
=== Combining semantic search with other queries
```

Reply:
Just a suggestion, feel free to ignore - I think this title would capture the "why" a bit better.